py/float.go

```diff
@@ -394,6 +394,21 @@ func (a Float) M__ge__(other Object) (Object, error) {
 	return NotImplemented, nil
 }
 
+// Properties
+func init() {
+	FloatType.Dict["is_integer"] = MustNewMethod("is_integer", func(self Object) (Object, error) {
+		if a, ok := convertToFloat(self); ok {
+			f, err := FloatAsFloat64(a)
+			if err != nil {
+				return nil, err
+			}
+			b := math.Abs(f - math.Round(f))
+			return NewBool(b < math.SmallestNonzeroFloat64), nil;
+		}
+		return nil, AttributeError
```

Comment:
````
go
return nil, cantConvert(a, "float")
````
might be better

Reply:
I'll change that part with cantConvert method.